dandi/dandiapi.py

```diff
@@ -404,6 +407,9 @@ def __init__(
             self._version = version
         self._data = data
 
+    def __str__(self) -> str:
+        return f"DANDI {self.identifier}/{self.version_id}"
```

Comment:
```suggestion
        return f"DANDI:{self.identifier}/{self.version_id}"
```
to mimic
```
$> curl --silent https://api.dandiarchive.org/api/dandisets/000003/versions/draft/ | jq .identifier     
"DANDI:000003"
```
but if we are to mint it ourselves (I guess to avoid fetching metadata which should have that `identifier`) -- do we have access to the instance name? then do not hardcode `DANDI` but use `instance_name.upper()`

Reply:
Done.